cernopendata/modules/fixtures/data/docs/delphi-news-first-release/delphi-news-first-release.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "author": "DELPHI Collaboration",
+    "body": {
+      "content": "delphi-news-first-release.md",
+      "format": "md"
+    },
+    "date_published": "2024-06-26",
+    "experiment": [
+      "DELPHI"
+    ],
+    "short_description": {
+      "content": "The DELPHI Collaboration has released the first set of its data samples for the Open Data Portal. Regardless if you are a researcher, teacher, student or just interested, start your journey through this exciting first dataset from LEP."
+    },
+    "featured": 1,
+    "slug": "delphi-news-first-release-2024",
```

Comment:
Note that the slug will uniquely identify the URL of the news item. It is not necessary to use news there, and dunno about "first release" which seems to hint that there will be more... We may want to model the slug after the title will be known.

Reply:
delphi-news-first-release-2024 -> delphi-data-release-2024 ?